build/scripts/main.js

```diff
@@ -21,10 +21,22 @@ Load the time and populate the DOM.
 function LoadTime()
 {
     var timeCurrent = new Date();
-    var timeHours = timeCurrent.getHours();
+    // Hour
+    var timeHoursRaw = timeCurrent.getHours();
+    var timeHoursDisplay = 0;
```

Comment:
Wouldn't this mean timeHoursDisplay would be 0 if timehoursraw was < 12, because you never set the value if timehoursraw < 12.

Reply:
Shit.